replica/distributed_loader.cc

```diff
@@ -38,6 +38,15 @@ static const std::unordered_set<std::string_view> system_keyspaces = {
                 db::system_keyspace::NAME, db::schema_tables::NAME
 };
 
+// Not super nice. Adding statefulness to the file. 
+static std::unordered_set<sstring> load_prio_keyspaces;
+static bool population_started = false;
+
```

Comment:
I expect it was removed exactly because of this (and the war on statics). Maybe we should make it nicer.

Reply:
My plan is to move it to extensions instead, but right now I would first like to make enterprise replicated encryption work again... 
So first, this revert series, then a series adding secondary way of prio (extension), then update ent to use this instead, then finally retire the above statics.

Reply:
Ok. /cc @xemul (who would otherwise remove it again)